src/OrbitGl/App.h

```diff
@@ -452,6 +452,9 @@ class OrbitApp final : public DataViewFactory, public orbit_capture_client::Capt
 
   [[nodiscard]] bool HasFrameTrackInCaptureData(uint64_t instrumented_function_id) const;
 
+  enum class SelectionMode { kFirst, kLast, kMin, kMax };
```

Comment:
I don't like the name of this `enum` and of the method too much, I think "selection" is too generic. If I think of selection I think more of a user clicking or dragging on a visual element, while here what really happens is a search, that results in moving to a different position in the capture, and finally a selection. I propose two alternatives:
- Rename this `enum` to something like `JumpToTextBoxMode` or `JumpToAndSelectTextBoxMode` or `FindAndSelectTextBoxMode`, and the method to something like `JumpToTextBoxAndZoom` or `JumpToAndSelectTextBox` or `FindAndSelectTextBox` ("find" is probably my favorite);
- Have four different methods instead of the switch, and call them something like `JumpToFirstTextBoxAndZoom`, etc., or `JumpToAndSelectFirstTextBoxAndZoom`, etc., or `FindAndSelectFirstTextBoxAndZoom`, etc.
What do you think?

Reply:
I agree. I will go for `JumpToTextBoxMode` and `JumpToTextBoxAndZoom`.

Reply:
Done.

Reply:
Great, thanks!